src_assets/common/assets/web/Navbar.vue

```diff
@@ -1,60 +1,93 @@
-                    <li class="nav-item">
-                        <a class="nav-link" href="/"><i class="fas fa-fw fa-home"></i> {{ $t('navbar.home') }}</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/pin"><i class="fas fa-fw fa-unlock"></i> {{ $t('navbar.pin') }}</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/apps"><i class="fas fa-fw fa-stream"></i> {{ $t('navbar.applications') }}</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/config"><i class="fas fa-fw fa-cog"></i> {{ $t('navbar.configuration') }}</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/password"><i class="fas fa-fw fa-user-shield"></i> {{ $t('navbar.password') }}</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/troubleshooting"><i class="fas fa-fw fa-info"></i> {{ $t('navbar.troubleshoot') }}</a>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </nav>
+  <nav class="navbar navbar-light navbar-expand-lg navbar-background header">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="/" title="Sunshine">
+        <img src="/images/logo-sunshine-45.png" height="45" alt="Sunshine">
+      </a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+              aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link" href="/"><i class="fas fa-fw fa-home"></i> {{ $t('navbar.home') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/pin"><i class="fas fa-fw fa-unlock"></i> {{ $t('navbar.pin') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/apps"><i class="fas fa-fw fa-stream"></i> {{ $t('navbar.applications') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/config"><i class="fas fa-fw fa-cog"></i> {{ $t('navbar.configuration') }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/password"><i class="fas fa-fw fa-user-shield"></i> {{
+                $t('navbar.password')
+              }}</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/troubleshooting"><i class="fas fa-fw fa-info"></i> {{
+                $t('navbar.troubleshoot')
+              }}</a>
```

Comment:
```suggestion
            <a class="nav-link" href="/password"><i class="fas fa-fw fa-user-shield"></i>
                {{ $t('navbar.password') }}</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="/troubleshooting"><i class="fas fa-fw fa-info"></i>
                {{ $t('navbar.troubleshoot') }}</a>
```

Having the i18n opening/closing brackets on separate lines is a bit confusing.

Reply:
CLion reformat got me there, I wasn't expecting this haha